src/pymmcore_widgets/_exposure_widget.py

```diff
@@ -4,34 +4,54 @@
 
 from pymmcore import g_Keyword_CoreCamera, g_Keyword_CoreDevice
 from pymmcore_plus import CMMCorePlus
-from qtpy import QtWidgets as QtW
 from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QApplication, QDoubleSpinBox, QHBoxLayout, QLabel, QWidget
 from superqt.utils import signals_blocked
 
 
-class ExposureWidget(QtW.QWidget):
-    """Generic widget to get/set exposure time on a camera."""
+class ExposureWidget(QWidget):
+    """A Widget to get/set exposure on a camera.
+
+    Parameters
+    ----------
+    camera : str
+        The camera device label. By default, None. If not specified,
+        the widget will use the current Camera device.
+    parent : Optional[QWidget]
+        Optional parent widget. By default, None.
+    mmcore: Optional[CMMCorePlus]
+        Optional `CMMCorePlus` micromanager core.
+        By default, None. If not specified, the widget will use the active
+        (or create a new) `CMMCorePlus.instance()`.
+
+    Examples
+    --------
+    !!! example "Combining `ExposureWidget` with other widgets"
+
+        see [ImagePreview](../ImagePreview#example)
+
+    """
 
     def __init__(
         self,
         camera: str = None,  # type: ignore
+        parent: Optional[QWidget] = None,
```

Comment:
also, let's undo all of these changes that move the `parent` argument in front of the star... don't like that